### ISSUE TITLE
Reviews [NIP-??] Positive Interoperability Tracker

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#136]
working spec: https://github.com/nostr-protocol/nips/pull/879

**app**|**see reviews*|**created reviews**|**comment**
-----|-----|-----|-----
coracle|?|?| @staab advised[ coracle implemented](https://github.com/nostr-protocol/nips/pull/879#issuecomment-1814464220)

**Comment by alltheseas** [comment]
@staab what are the use cases in Coracle for reviews? For instance relay reviews? Marketplace vendor reviews?

**Comment by staab** [comment]
Relay reviews only right now. This is a pretty stale feature, and there's not that much data associated with it, so don't consider my implementation canonical.

**Comment by alltheseas** [comment]
Has anyone else implemented?

**Comment by alltheseas** [comment]
@calvadev @gzuuus @vitorpamplona do yall plan adding reviews NIP in your marketplace apps/segments?

**Comment by calvadev** [comment]
> @calvadev @gzuuus @vitorpamplona do yall plan adding reviews NIP in your marketplace apps/segments?

@alltheseas Yes; actually about to wrap up product/merchant reviews based on the above NIP-85 PR. Will add a comment to the PR when done.

**Comment by vitorpamplona** [comment]
> do yall plan adding reviews NIP in your marketplace apps/segments?

Yes, I am just waiting for the NIP

**Comment by calvadev** [comment]
> > @calvadev @gzuuus @vitorpamplona do yall plan adding reviews NIP in your marketplace apps/segments?
> 
> @alltheseas Yes; actually about to wrap up and deploy product/merchant reviews based on the above NIP-85 PR within the next week or so. Will add a comment to the PR with the "spec".

Here's the comment:

https://github.com/nostr-protocol/nips/pull/879#issuecomment-2502210146

**Comment by gzuuus** [comment]
Yes, will implement them also, not sure when, but would be nice to have a solid spec for them

**Comment by calvadev** [comment]
> > @alltheseas Yes; actually about to wrap up and deploy product/merchant reviews based on the above NIP-85 PR within the next week or so. Will add a comment to the PR with the "spec".
> 
> Here's the comment:
> 
> [nostr-protocol/nips#879 (comment)](https://github.com/nostr-protocol/nips/pull/879#issuecomment-2502210146)

@alltheseas added support for it on shopstr today
